system/puppet.py

```diff
@@ -147,6 +156,7 @@ def main():
             facter_basename=dict(default='ansible'),
             environment=dict(required=False, default=None),
             certname=dict(required=False, default=None),
+            tags=dict(required=False, default=None),
```

Comment:
No worries, tags for tasks and tags for puppet can be mixed as they are not on the same level, and btw there are other modules already have tags args.

~~~yaml
- puppet:
    tags:
      - update
      - nginx
  tags:
    - puppet
~~~